backend/src/test/java/gov/cdc/usds/simplereport/db/model/PersonSerializationTest.java

```diff
@@ -75,14 +74,14 @@ void serialize_withFacility_noOrgOrFacility() throws IOException {
         new DeviceSpecimenType(
             new DeviceType("Bill", "Weasleys", "1", "12345-6", "E"),
             new SpecimenType("Troll Bogies", "000111222"));
-    StreetAddress addy =
-        new StreetAddress(Collections.singletonList("Moon Base"), "Luna City", "THE MOON", "", "");
+    StreetAddress address =
+        new StreetAddress("736 Jackson PI NW", null, "Washington", "DC", "20503", "Washington");
```

Comment:
🤣 🚀  Well, fake data gets you only so far...

But actually this change confuses me, because the validity of the data is irrelevant here and I'm just going to and from a string or file—did you just do a sweep for `new StreetAddress` and mercilessly replace them with something valid?

Reply:
Good flag. I was meaning to remove these updates.

> did you just do a sweep for new StreetAddress and mercilessly replace them with something valid?

Yes, I was hoping to update everything to use the data factory. That didn't work for these tests as it turns out they  are asserting behavior about the address

```
   assertThat(serialized)
        .extractingJsonPathArrayValue("address.street")
        .hasSize(3)
.isEqualTo(List.of("1600 Pennsylvania Avenue", "Right Side Door", "Down the Steps"));


```